.github/ISSUE_TEMPLATE.md

```diff
@@ -1 +1,26 @@
 **Important**: This is a public repository. Anyone in the world can see what's posted here. If you are posting screenshots or log files, please **carefully examine them for** the presence of any kind of **protected health information** (PHI). Images or logs containing PHI _must_ be posted in fully-redacted form, with no visible PHI.
+
+---
+
+If you're reporting a bug copy the following template below, replacing the _italicised_ text with your own. This is important to ensure we can look at your bug quickly and efficiently, and will improve the speed in which it gets fixed :-)
+
+---
+
+_Brief description of bug encountered_.
+
+**Steps to reproduce**:
+ - _Write individual steps here_
+ - _That can be used to reproduce_
+ - _The problem_
+
+**What should happen**:
+ - _Detail what you **expect** to happen when you complete the last step_
+
+**What actually happens**:
+ - _Describe what happens (the bug!)_
+
```

Comment:
Have a section for logs - people always forget to include the logs. Also list the places the logs are found (browser console, gardener, nginx, couch, etc).

Reply:
I would make sure to document _how_ to get logs, either in the issue template or link to relevant documentation. Also, not all those reporting bugs would have access to logs... so we should request the URL and whoever is investigating can access the logs.